moveit_ros/visualization/motion_planning_rviz_plugin/src/motion_planning_frame.cpp

```diff
@@ -515,7 +548,7 @@ void MotionPlanningFrame::importResource(const std::string& path)
     }
     else
     {
-      QMessageBox::warning(this, QString("Import error"), QString("Unable to import scene"));
+      QMessageBox::warning(this, QString("Import error"), QString("Unable to import 3D object"));
```

Comment:
All objects in our context are 3D.
```suggestion
      QMessageBox::warning(this, QString("Import error"), QString("Unable to import object"));
```

Reply:
Done